src/components/Buttons/TranslateButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '@/components/ui/button';
+import { useTranslation } from 'react-i18next';
+
+// Header에서 언어를 바꿀 수 있는 버튼입니다.
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  className?: string;
+  toggleLanguage?: () => void;
```

Comment:
```suggestion
  onToggleLanguage?: () => void;
```

Event니까 `on` 접두사를 붙여줍시다.

Reply:
toggleLanguage -> onToggleLanguage 변경 완료

질문사항입니다.
src/container/common/Header.tsx에서
```
  // 언어 변경 함수
  const toggleLanguage = () => {
    const newLang = i18n.language === 'ko' ? 'en' : 'ko';
    i18n.changeLanguage(newLang);
    localStorage.setItem('lang', newLang);
  };
```
요 바깥 함수는 handdle 접두사로 붙여주는게 좋나요??